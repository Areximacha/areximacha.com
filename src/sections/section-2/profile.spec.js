import React from 'react'
import renderer from 'react-test-renderer'

import Profile from './profile'

describe('<Profile/>', () => {
  describe('@render', () => {
    it('renders correctly', () => {
      const tree = renderer.create(<Profile />).toJSON()

      expect(tree).toMatchSnapshot()
    })
  })
})
