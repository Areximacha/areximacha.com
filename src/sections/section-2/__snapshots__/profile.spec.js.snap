// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Profile/> @render renders correctly 1`] = `
<section
  className="css-1itll8f"
  id="profile"
>
  <div
    className="css-194vaoi"
  >
    <h2>
      Coder. Gamer. Geek.
    </h2>
    <p>
      <strong>
        Jay Tang
      </strong>
       - A self confessed nerd, obsessed with video games and possessor of an encyclopedic knowledge of comic book history.
    </p>
    <p>
      I am a web technologist, currently specialising in front-end engineering with React, Redux, NodeJS and a bunch of other stuff.
    </p>
    <p>
      I love building things and I am always excited to play with new tech, solving problems and leveling up my abilities to become more awesome.
    </p>
  </div>
</section>
`;
